### PR TITLE
update banner to: Neon 🤝 Vercel: Create a branch for every Preview Deployment

### PR DESCRIPTION
**Pull request by raoufchebri**
https://neon-next-a30jzwfos-neondatabase.vercel.app/

### CODE REVIEW

File: src/components/shared/topbar/topbar.jsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import ArrowRightIcon from 'icons/arrow-right.inline.svg';
 const TopBar = () => (
   <Link
     className="safe-paddings relative z-40 flex h-11 w-full items-center justify-center bg-primary-1 px-4 py-3 leading-none transition-colors duration-200 hover:bg-[#1AFFB2] xs:h-auto"
-    to="/developer-days-1"
+    to="/neon-vercel-integration"
   >
     <span className="mr-4 truncate border-r border-black border-opacity-20 py-1 pr-4 text-sm font-semibold">
-      Watch Neon Developer Days 🚀
+      Neon 🤝 Vercel: Create a branch for every Preview Deployment
```
Comment (raoufchebri) [inline src/components/shared/topbar/topbar.jsx:12]:
```suggestion
      Create a Neon branch for every Vercel Preview Deployment
```

File: src/components/shared/topbar/topbar.jsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import ArrowRightIcon from 'icons/arrow-right.inline.svg';
...
     className="safe-paddings relative z-40 flex h-11 w-full items-center justify-center bg-primary-1 px-4 py-3 leading-none transition-colors duration-200 hover:bg-[#1AFFB2] xs:h-auto"
-    to="/developer-days-1"
+    to="/neon-vercel-integration"
   >
     <span className="mr-4 truncate border-r border-black border-opacity-20 py-1 pr-4 text-sm font-semibold">
-      Watch Neon Developer Days 🚀
+      Create a Neon branch for every Vercel Preview Deployment
     </span>
     <span className="inline-flex items-center text-sm font-bold xs:hidden">
       <span>Read More</span>
```
Comment (raoufchebri) [inline src/components/shared/topbar/topbar.jsx:15]:
```suggestion
      <span>Learn More</span>
```

**Review by danieltprice — APPROVED** [review]
lgtm

**Review by m-abdelwahab — APPROVED** [review]
ship it 🚀

**Comment by m-abdelwahab** [comment]
Or actually what if make it 

"Neon integration on Vercel: Create a Neon branch for every Preview Deployment"
Arc-001361

File: src/components/shared/topbar/topbar.jsx [hunk 3]
```diff
@@ -6,13 +6,13 @@ import ArrowRightIcon from 'icons/arrow-right.inline.svg';
 const TopBar = () => (
   <Link
     className="safe-paddings relative z-40 flex h-11 w-full items-center justify-center bg-primary-1 px-4 py-3 leading-none transition-colors duration-200 hover:bg-[#1AFFB2] xs:h-auto"
-    to="/developer-days-1"
+    to="/neon-vercel-integration"
   >
     <span className="mr-4 truncate border-r border-black border-opacity-20 py-1 pr-4 text-sm font-semibold">
-      Watch Neon Developer Days 🚀
+      Create a Neon branch for every Vercel Preview Deployment
```
Comment (raoufchebri) [inline src/components/shared/topbar/topbar.jsx:12]:
```suggestion
      Neon integration on Vercel: Create a branch for every Preview Deployment
```

**Review by m-abdelwahab — APPROVED** [review]

**Pull request merged**